x-pack/plugins/fleet/server/services/api_keys/logstash_api_keys.ts

```diff
@@ -58,6 +59,7 @@ export async function generateLogstashApiKey(esClient: ElasticsearchClient) {
               'synthetics-*-*',
               '.logs-endpoint.diagnostic.collection-*',
               '.logs-endpoint.action.responses-*',
+              'profiling-*',
```

Comment:
```suggestion
              'profiling-*-*',
```

Let's match our other data stream pattern conventions here too (I assume the profiling data streams will be using this convention)